src/sentry/static/sentry/app/utils/dates.tsx

```diff
@@ -212,3 +212,8 @@ export function parsePeriodToHours(str: string): number {
       return -1;
   }
 }
+
+export const use24Hours = (): boolean => {
+  const user = ConfigStore.get('user');
+  return user && user.options && user.options.clock24Hours;
```

Comment:
We can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here

```suggestion
  return !!user?.options?.clock24Hours
```

Reply:
Oh dang that's nice!